examples/Demo/FluentUI.Demo.Client/Documentation/Components/Spacing/Spacing.md

```diff
@@ -144,3 +144,26 @@ These values can be customized or overridden in your own CSS files.
 --spacingHorizontalXXXL:  28px;
 --spacingHorizontalXXXXL: 32px;
 ```
+
+## Appendix - Boostrap
+
+If you use **Bootstrap** (or **MudBlazor**) in your project, we use the same class names
+(for consistency and ease of use). In this case, you choice to use **Bootstrap** or **FluentUI Blazor**
+classes for margins and paddings. You need to check the loading order of the **Styles** files in your main page:
+
+For a project named **FluentUI.Demo**, in this example the Boostrap stylesheet is loaded after the FluentUI stylesheet.
+So, the **Bootstrap** classes override the **FluentUI Blazor** classes.
+**Bootstrap** will therefore be used for margins and paddings.
```

Comment:
For example this project below, named **FluentUI.Demo**, the Boostrap stylesheet is loaded after the Fluent UI stylesheet. So, the **Bootstrap** classes override the **FluentUI Blazor** classes. The **Bootstrap** values will be used for margins and paddings.